packages/launchdarkly-adapter/modules/adapter/adapter.spec.js

```diff
@@ -68,12 +68,16 @@ describe('when configuring', () => {
 
   describe('with user key', () => {
     beforeEach(() => {
-      return adapter.configure({
-        clientSideId,
-        user: userWithKey,
-        onStatusStateChange,
-        onFlagsStateChange,
-      });
+      return adapter.configure(
```

Comment:
See types.

TL;DR: the `configure` function accepts the event handlers as a second argument.